src/Symfony/Bridge/Monolog/Tests/Processor/WebProcessorTest.php

```diff
@@ -36,10 +36,6 @@ public function testUsesRequestServerData()
 
     public function testCanBeConstructedWithExtraFields()
     {
-        if (!$this->isExtraFieldsSupported()) {
-            $this->markTestSkipped('WebProcessor of the installed Monolog version does not support $extraFields parameter');
-        }
```

Comment:
You can then also remove the `isExtraFieldsSupported()` method itself. However, doing this change requires to bump the dependency on `monolog/monolog` to `1.11.0` or higher (prior versions did not support extra fields in the web processor).